sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/test/scala/com/azure/cosmos/spark/CosmosConfigSpec.scala

```diff
@@ -49,6 +53,16 @@ class CosmosConfigSpec extends UnitSpec {
   }
 
   it should "parse read configuration" in {
+    val userConfig = Map(
+      "spark.cosmos.read.forceEventualConsistency" -> "false"
```

Comment:
could you please add a test to capture the default value if this config is not present?

Reply:
Done